<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>puPluginDockWidgetBase</class>
 <widget class="QDockWidget" name="puPluginDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PU Plugin</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="importVfkLayout">
     <item>
      <layout class="QHBoxLayout" name="vfkFileLayout">
       <item>
        <widget class="QLabel" name="vfkFileLabel">
         <property name="text">
          <string>VFK soubor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vfkFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="vfkFileBrowseButton">
         <property name="text">
          <string>Procházet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="loadVfkLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="loadVfkFileLayout">
       <item>
        <widget class="QProgressBar" name="loadVfkFileProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadVfkFileButton">
         <property name="text">
          <string>Načíst</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
